### DISCUSSION TITLE
Kura reference devices

### DISCUSSION THREAD

**Discussion by pierantoniomerlino** [eclipse-kura/kura#5226] [Ideas]
Hello everyone!

As discussed [here](https://github.com/eclipse/kura/discussions/5224), Kura 5.5 will drop some profiles for specific hardware and it slowly moves towards supporting only **generic** profiles tailored on the architecture instead of a device.

However, we want to have some "reference" hardware devices that will be used for testing and development. The idea is to support one or more device for each architecture. A proposal is presented below:

Device | Architecture | OS | notes |
-------|------|-----|----|
Raspberry Pi 4/5 | arm32 | RaspberryPi OS (32bit) | 
Raspberry Pi 4/5 | arm64 | RaspberryPi OS | 
Nvidia Jetson AGX Orin | arm64 | Ubuntu 22-04 |
ZimaBoard/Blade | x86_64 | Ubuntu | https://www.zimaspace.com/

Any idea? Are there any other board/device/devkit that are "popular" to be used as a reference?

**Comment by alexcorvis84** [comment]
What about adding [Radxa](https://radxa.com/) or [Banana Pi](https://www.banana-pi.org/) alternatives for the RasberryPi ones?
